pinot-controller/src/test/java/org/apache/pinot/controller/api/AccessControlTest.java

```diff
@@ -19,49 +19,40 @@
 package org.apache.pinot.controller.api;
 
 import java.io.IOException;
-import java.util.Map;
-
-import org.apache.pinot.controller.ControllerConf;
 import org.apache.pinot.controller.api.access.AccessControl;
 import org.apache.pinot.controller.api.access.AccessControlFactory;
-import org.apache.pinot.controller.helix.ControllerTest;
 import org.testng.Assert;
 import org.testng.annotations.AfterClass;
 import org.testng.annotations.BeforeClass;
 import org.testng.annotations.Test;
 
+import static org.apache.pinot.controller.ControllerTestUtils.*;
 
-public class AccessControlTest extends ControllerTest {
 
-  @BeforeClass
-  public void setUp() {
-    startZk();
-    
-    Map<String, Object> properties = getDefaultControllerConfiguration();
-    properties.put(ControllerConf.ACCESS_CONTROL_FACTORY_CLASS, DenyAllAccessFactory.class.getName());
+public class AccessControlTest {
+
+  private static final String TABLE_NAME = "accessTestTable";
 
-    startController(properties);
+  @BeforeClass
+  public void setUp() throws Exception {
+    validate();
```

Comment:
This is somewhat confusing to read - why does setup() call something named validate(), instead of setting up the cluster? One has to go to inside validate() to figure out what is going.

Perhaps it can be named `setupClusterAndValidate()`, or something more readable?

Reply:
Fixed.